vcmi.spec

```diff
@@ -43,8 +43,14 @@ BuildRequires:  minizip-ng-devel
 BuildRequires:  tbb-devel
 BuildRequires:  zlib-devel
 BuildRequires:  ffmpeg-devel
+%if 0%{?fedora} == 39
```

Comment:
It should be >= so it can handle newer fedora releases (rawhide is currently at 40).

Once fixed (and tested) feel free to apply.

Reply:
Then you are wrong twice ;)
You are expected to land newer features in the most recent branch. So please verify rawhide, then f39